### DISCUSSION TITLE
BingAI, and ChatGPT Browser

### DISCUSSION THREAD

**Discussion by tttll1091** [danny-avila/LibreChat#255] [Q&A]
- Does BingAI use the OpenAI API Langchain? Or are you using BingAI's API?

- Does ChatGPT Browser use the free version of OpenAI API? Are you aware that this is not the WEB version of ChatGPT?

**Comment by danny-avila** [comment]
Both use reverse engineered access of their respective websites' API

**Reply by tttll1091** [reply]
thanks
